### ISSUE TITLE
Test the live and dev sites for accessibility

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/311-data#415]
### Overview
There are more than 100 accessibility testing tools.  Figuring out which ones to use can be a black hole.  For guidance we recommend this article:  [Which accessibility testing tool should you use?](https://medium.com/pulsar/which-accessibility-testing-tool-should-you-use-e5990e6ef0a)

### Summary of Article
The author recommends using the tools in the following order fixing as you go along, since no one tool catches all the relevant issues

[aXe](https://www.deque.com/axe/)
[SiteImprove](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc)
[Tenon](https://tenon.io/)
[WAVE](https://wave.webaim.org/extension/)
[Lighthouse](https://developers.google.com/web/tools/lighthouse/#devtools)

But if you want to test your site with other tools, here is a bigger [list](https://www.w3.org/WAI/ER/tools/)

### Action items
- [ ] Install the plugins
- [ ] Test the site against each plugin,  writing up a report of what the plugins consider to be inaccessible.  Use screenshots where necessary.  Put that info in this issue, either as an attached doc, or directly into the issue.

### Resources
From the HfLA Getting Started repo readme: [ADA Compliance Guide](https://www.ada.gov/pcatoolkit/chap5toolkit.htm). All web and mobile app projects at Hack for LA should be ADA compliant. We are an inclusive organization by charter.

###
Site to test: https://hackforla.github.io/311-data/

**Comment by ExperimentsInHonesty** [comment]
@itsjoycelee I have assigned this to Kevin, not for him to test everything, but for him to get familiar with the browser plugins
